config/default/config_ignore.settings.yml

```diff
@@ -1,10 +1,11 @@
 _core:
   default_config_hash: UVH1aJ4b44UM-VdPVN7hNNuuVqfReJxwfVeDQH1Hvsk
 ignored_config_entities:
-  - devel.toolbar.settings
-  - devel.settings
   - fathom_analytics.settings
-  - system.menu.devel
   - system.site
```

Comment:
Could be `system.site:page.front` perhaps?

Reply:
Changed to system.site:page to catch other pages

Reply:
Changed to system.site:page to catch other pages

Reply:
Good thinking!